### PR TITLE
feat: add test to verify risk parameters ranges

### PR DESCRIPTION
**Pull request by Jiajia-Cui**

### CODE REVIEW

File: core/integration/features/verified/risk-parameters-sigma-test.feature [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      | party2 | USD   | ETH/MAR0  | 5884     | 4999994116                  | 0        |
+
+    # mentainance margin level for LP: 181819*100*3.5569036=6.47e7
+    # initial  margin level for LP: 181819*100*3.5569036 *1.2=9.7e7
+
+    And the parties should have the following margin levels:
+      | party  | market id | maintenance | search   | initial  | release   |
+      | party0 | ETH/MAR0  | 64671266    | 77605519 | 97006899 | 109941152 |
+
+# Scenario: 003, test market ETH/MAR54(sigma=100),
```
Comment (jeremyletang) [inline core/integration/features/verified/risk-parameters-sigma-test.feature:141]: is that commented in purpose?

Comment (Jiajia-Cui) [reply core/integration/features/verified/risk-parameters-sigma-test.feature:141]: yes, I got the following error message when sigma is greater than 52, so I am going to change the validated range of sigma from 100 to 50 

2022-11-02T15:52:30.600Z	INFO	execution	price/statevar.go:58	no reference price available for market - cannot calculate price ranges	{"event-id": "20191130_000002"}
the opening auction period ends for market "ETH/MAR54" FATAL: some mock assertion failed: Cannot create a Decimal from NaN

File: core/integration/features/verified/risk-parameters-sigma-test.feature [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      | party2 | ETH/MAR53 | sell | 10     | 11    | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-14 |
+
+    When the opening auction period ends for market "ETH/MAR53"
+    And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/MAR53"
+    # And the network moves ahead "1" blocks
+
+    And the market data for the market "ETH/MAR53" should be:
+      | mark price | trading mode            | horizon | min bound | max bound | target stake | supplied stake | open interest |
+      | 10         | TRADING_MODE_CONTINUOUS | 43200   | 1         | 211       | 9999990      | 100000000      | 1             |
+    # target_stake = mark_price x max_oi x target_stake_scaling_factor x rf_short = 1 x 10 x 1 x 999999.00000000000 =557872
```
Comment (Galen-Cao) [inline core/integration/features/verified/risk-parameters-sigma-test.feature:69]: The comment of calculating target_stake has a typo in the result. Anything else looks good to me.

**Review by Galen-Cao — APPROVED** [review]
LGTM

**Pull request merged**